Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.7.3
```

Comment:
should this file be at the top-level? intuitively it would seem that it is meant to create docker image for the repository. Instead, it is a helper dockerfile. Maybe put it in (gulp) hack? Or what Tim suggested - `build`?